### DISCUSSION TITLE
ERROR EN LAB 04.02

### DISCUSSION THREAD

**Discussion by Jehb1998** [rramosp/ai4eng.v1#109] [04 - GESTIÓN DE DATOS]
Buenas tardes, estoy presentando un inconveniente con el laboratorio building dataset, a pesar de que estoy entregando la lista solicitada, el calificador me lo pone con nota de 0. Esto no me he permitido avanzar con la terminación de esta tarea y la siguiente. Agradecería cualquier aclaración
![image](https://user-images.githubusercontent.com/98557472/151442493-701648c9-44d6-4851-a792-281ff01bc7d0.png)

**Comment by Felipe-RA** [comment]
Saludos Jehb1998

Para poder revisar tu caso te agradecería que me pases por el correo tu notebook para mirarlo.

Quedo atento, hasta pronto
